admin-dev/themes/new-theme/public/sql_manager.bundle.js

```diff
@@ -1,6 +1,974 @@
-/******/!function(n){function t(r){if(e[r])return e[r].exports;var o=e[r]={i:r,l:!1,exports:{}};return n[r].call(o.exports,o,o.exports,t),o.l=!0,o.exports}// webpackBootstrap
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
your assets are in dev prod, some contain conflicts, and some are in conflict with develop branch anyway ^^
I think you should rebase and regenerate them